### ISSUE TITLE
Allow setting a default environment

### ISSUE THREAD

**Issue by nmss** [usebruno/bruno#2170]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

Currently when a collection have environments, by default `no environment` is selected

I wish I could assign a default one, so the at the collection I load is working without any more action

### Mockups or Images of the feature

![image](https://github.com/usebruno/bruno/assets/1228204/7f84f416-51eb-4dd5-ae5a-75b3306742f4)

**Comment by zagorec92** [comment]
I've already played around with this, so if you don't have the time/desire/etc. to implement it by yourself, I can take over IF it's not a problem.

What I've done so far:
- in environment list, default env is marked as bold
  - wanted to emphasize it while keeping the same UI layout
- Set as Default at the bottom
  - not sure is there any other position where this would be appropriately placed
- default environment is saved in collection's _bruno.json_ file

![image](https://github.com/usebruno/bruno/assets/4306892/f99148ac-b3ca-49dc-87ef-3ab6a38220f9)

**Comment by grubersjoe** [comment]
Seems like a duplicate of https://github.com/usebruno/bruno/issues/1598

**Comment by zagorec92** [comment]
> Seems like a duplicate of #1598

Yep, makes more sense, it's pretty much what postman does with remembering selected environment.

**Comment by satokenta940** [comment]
This is indeed very convenient, and [Apidog ](https://www.apidog.com/?utm_source=&utm_medium=blogger&utm_campaign=test1)supports it, so the experience will be better.
